kolibri/core/exams/models.py

```diff
@@ -52,6 +52,31 @@ class Exam(AbstractFacilityDataModel):
     """
     The `question_sources` field contains different values depending on the 'data_model_version' field.
 
+    V3:
+        Represents a list of questions of V2 objects each of which are now a "Exam/Quiz Section"
+        and extends it with an additional description field
+
+        # Exam
+        [
+            # Section 1
+            {
+                  "section_title": <section title>,
+                  "description": <section description>,
```

Comment:
Sorry, I totally missed this when reviewing the spec in the issue - when I last looked at the designs, each section was able to be independently set as fixed order/random order, iirc? Presumably, then the top level `learners_see_fixed_order` value for the quiz now pertains to the sections, so we would need an equivalent field within the sections.